quadriga/src/main/java/edu/asu/diging/quadriga/elements/factory/IActorFactory.java

```diff
@@ -0,0 +1,11 @@
+package edu.asu.diging.quadriga.elements.factory;
+
+import edu.asu.diging.quadriga.model.elements.Actor;
+
+public interface IActorFactory {
```

Comment:
either all factories should be deprecated or none. I don't think we will need them since they don't do anything really besides creating a new instance. So, I say let's make them all deprecated.